src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -91,6 +91,18 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
             OptionSet options = null,
             CancellationToken cancellationToken = default);
 
+        internal virtual async Task<(CompletionList completionList, bool expandItemsAvailable)> GetCompletionsInternalAsync(
```

Comment:
>expandItemsAvailable [](start = 73, length = 20)

Reviewing the idea of CompletionList, I can assume that          expandItemsAvailable is worth to add there.

Reply:
I remember there's cases where we return null as CompletionList but still need to know if expand items are available or not

Reply:
Thank you for explanations!

---
In reply to: [296050156](https://github.com/dotnet/roslyn/pull/36146#discussion_r296050156) [](ancestors = 296050156)